study.md

```diff
@@ -237,7 +242,10 @@ Are heuristics the same thing rules? What are some design heuristics you can use
 to improve the design of your code?
 
 ```md
-<!-- your answer here -->
+Basically, yes, heuristics should be considered rules. The sooner you do this, the sooner
+your code will become a functioning thing of beauty. Sandi Metz rules for developers are
+basically our 4 commandments. Also going slow and being meticulous about heuristics is
+an essential heuristic itself.
```

Comment:
Yep, but heuristics aren't laws. You can break the rules of the thumb if you have a reason. But you probably shouldn't. Yet. 😄